src/radio/index.js

```diff
@@ -139,8 +139,14 @@ class Radio extends EventEmitter {
 
     if (!this.order.some(u => u.equals(user))) return false;
 
-    this.skips.add(user.id);
+    if (this.skips.has(user.id)) {
+      this.skips.delete(user.id);
+    } else {
+      this.skips.add(user.id);
+    }
+
     this.checkSkips();
+    this.emit('skips', this.skips);
```

Comment:
checkSkips may advance the song; does this have the potential to emit a useless skips event after a song gets skipped?

Reply:
Good point. I suppose I should put it prior to the checkSkips call, especially considering when we had the emit in check skips it was doing it prior to moving to the next song as well.